spec/helpers/application_helper/toolbar_chooser_spec.rb

```diff
@@ -173,4 +173,38 @@
       end
     end
   end
+
+  describe '#center_toolbar_filename_classic' do
+    subject { chooser.send(:center_toolbar_filename_classic) }
+
+    {
+      "manageiq/providers/embedded_ansible/automation_manager/playbook"            => "playbook",
+      "manageiq/providers/embedded_automation_manager/configuration_script_source" => "repository",
+      "manageiq/providers/embedded_automation_manager/authentication"              => "credential"
+    }.each do |layout, screen|
+      context "toolbar for summary screen of Ansible #{screen}" do
+        let(:chooser) { ApplicationHelper::ToolbarChooser.new(nil, nil, :display => "main", :lastaction => "show", :layout => layout) }
```

Comment:
where this chooser is being used?

Reply:
Is this what you are looking for? https://github.com/ManageIQ/manageiq-ui-classic/pull/3721/files#diff-e749e1716139d3f230dd592c70c3099bR178